client/src/components/pages/Completed/index.jsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useCallback } from 'react' // 追加
+import { useRecoilValue, useSetRecoilState } from 'recoil' // 追加
+import { axios } from '../../../utils/axiosConfig' // 追加
+import { todoState, completedTodoListState } from '../../../stores/todoState'
+
+import { Layout } from '../../ui/Layout'
+import { ListItem } from '../../ui/ListItem'
+
+import { errorToast } from '../../../utils/errorToast' 
+
+import styles from './index.module.css'
+
+export const Completed = () => {
+  const todos = useRecoilValue(completedTodoListState)
+  const setTodos = useSetRecoilState(todoState) 
+
+  const handleToggleButtonClick = useCallback(
+    (id) => {
+      axios
+        .patch(`http://localhost:3000/todo/${id}/completion-status`, {
+          isCompleted: todos.find((todo) => todo.id === id).isCompleted,
+        })
+        .then(({ data }) => {
+          setTodos((prev) =>
+            prev.map((todo) => (todo.id === data.id ? data : todo))
+          )
```

Comment:
[nits]
不要な括弧がいくつかあるので、以下のように簡略化できますね！
```suggestion
          setTodos(prev => prev.map(todo => todo.id === id ? data : todo))
```

Reply:
二つ目の指摘と同様に、ほかにも不要な()がないか確認して修正します。